### PR TITLE
Add support for `universe/ids` POST endpoint

### PR DESCRIPTION
**Pull request by pilot538**
Added support for `universe/ids` POST endpoint. 

Improvements could be made to pull the categories from the SDE directly, versus manually coding them here. 

Also very open to a better solution, this is just the one I came up with.

### CODE REVIEW

**Comment by Celeo** [comment]
I pushed a commit to resolve the merge conflicts.

File: src/groups/universe.rs [hunk 1]
```diff
@@ -205,4 +224,14 @@ impl<'a> UniverseGroup<'a> {
...
     );
+
+    api_post!(
+        /// Get IDs from a list of names
+        get_ids,
+        "post_universe_ids",
+        RequestType::Public,
+        Ids,
+        ,
+        names: &Vec<String>,
```
Comment (Celeo) [inline src/groups/universe.rs]: Let's try making this a more permissive type:

```suggestion
names: &[&str],
```

Does that work for your code?

**Comment by Celeo** [comment]
I fixed the formatting; I think I messed up the replacement suggestion indent.

**Comment by Celeo** [comment]
Do me a favor and confirm that new signature works for you, and then I'll get it merged.

**Comment by pilot538** [comment]
Yes, this works for me. Thanks!

**Comment by Celeo** [comment]
Super, thank you for the contribution!

**Comment by Celeo** [comment]
Released in [0.32.0](https://github.com/Celeo/rfesi/releases/tag/v0.32.0).

**Pull request merged**